src/smart-components/user/user-table-helpers.tsx

```diff
@@ -29,47 +31,60 @@ export interface RowProps {
     CellObject // status
   ];
   selected: boolean;
+  advancedPermissions?: boolean;
```

Comment:
Let's rename it to `authModel`
```suggestion
  authModel?: boolean;
```

Reply:
Fixed